en/news/_posts/2015-03-03-ruby-2-2-1-released.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: news_post
+title: "Ruby 2.2.1 Released"
+author: "hsbt"
+translator:
+date: 2015-03-03 03:00:00 +0000
+lang: en
+---
+
+We are pleased to announce the release of Ruby 2.2.1.
+This is the first tiny version release of the stable 2.2 series.
```

Comment:
@makimoto Yup. We use a slightly different semantic versioning, please see: https://www.ruby-lang.org/ko/news/2013/12/21/ruby-version-policy-changes-with-2-1-0/.